README.md

```diff
@@ -852,6 +852,150 @@ TODO-A: - TBD
 TODO-B: Problems: None
 ```
 
+
+# Re-Reviewing an approved package
+
+Right now this is **not active**, it is - so far - a documentation of the intent to do so helping all involved parties to better understand what this would mean.
+
+The MIR team and thereby in extension Ubuntu continuously evolves the MIR ruling, for example we add rules based on painful lessons learned to avoid the same to happen again. In that regard some asks for testing, quality and such might be best practise now, but have not been fulfilled by packages that are in main for a long time.
```

Comment:
```suggestion
The MIR team and thereby in extension Ubuntu continuously evolves the MIR ruling. For example, we add rules based on painful lessons learned to avoid the same from happening again. In that regard the demand for testing, quality and so on might be best practice now, but have not been fulfilled by packages that are in main for a long time.
```

typo: `practise` -> `practice`
suggestion: slightly different punctuation/wording for better reading.

Reply:
Integrated, thanks!